include/controller.h

```diff
@@ -59,9 +65,12 @@ class Ctrl{
     bool _abort_calculation; // variable for main thread to signal stopping the calculation
     bool _calculation_finished;
     bool _to_gui = true; // determines whether to print to console or to GUI
-    
+    bool _quiet = true; // silences all non-result command line outputs
+   
+    void displayInput(CalcReportBundle&, const unsigned=mvOUT_ALL);
+    void displayResults(CalcReportBundle&, const unsigned=mvOUT_ALL);
+    void displayCavityList(CalcReportBundle&, const unsigned=mvOUT_ALL);

```

Comment:
On windows, in command line mode, the column titles are not aligned with the values. If it is the case on Unix too, I would suggest to put the column titles over two lines like in the report.
```
Cavity	Occupied	Accessible	Cavity center coordinates (A)
ID	Volume (A^3)	Volume (A^3)	x	y	z
1	2426.27		1907.44		6.9	6.9	7
2	24.467		0.862		6.3	6.3	6.3
```

Reply:
I've made some changes to address this. It works fine now, however, it's not so perfect in unicode mode. Since that's not the default anyway I'll leave it as is for now.